src/rotation/Rotation.c

```diff
@@ -0,0 +1,74 @@
+#include "Rotation.h"
+#include <stdlib.h>
+#include <math.h>
+
+
+int RotatePxlX(float angle, int x, int xc, int y, int yc);
+int RotatePxlY(float angle, int x, int xc, int y, int yc);
+
+int Rotate(Image* BitMap, float angle)
+{
+    angle = angle *(float)(3.1415/180);
+    int xc = BitMap->width/2;
+    int yc = BitMap->height/2;
+    Pixel** BitMapRotate = malloc(BitMap->height * sizeof (*BitMapRotate));
```

Comment:
why not just use `sizeof(Pixel)` instead of `sizeof(*BitMapRotate)`? 
It is more clear and less prone to errors

Reply:
It's the same (and also the code can be reuse easily)